components/forms/form-builder/form-sections/ImportantText.tsx

```diff
@@ -93,24 +93,30 @@ const unresolvedDecsText =
   "If you know of any UNRESOLVED Significant Events, Complaints, Other investigations since your last ARCP/RITA/Appraisal, please provide a brief summary, including where you were working, the date of the event, and your reflection where appropriate. If known, please identify what investigations are pending relating to the event and which organisation is undertaking the investigation.";
 
 const ltftDiscussionText1 =
-  "Please provide the contact details of the Training Programme Director (TPD) you have discussed your Changing hours (LTFT) proposals with prior to completing this form. ";
+  "Please provide the contact details of the Training Programme Director (TPD) you have discussed your Changing hours (LTFT) proposal with prior to completing this form. ";
 
 const ltftDiscussionText2 =
-  "When you submit this appication, your TPD will receive an email which will include a summary of your proposals.";
+  "When you submit this application, your TPD will receive an email which will include a summary of your proposal.";
```

Comment:
When going through a single LTFT application, I was unclear why 'proposals' was in the plural. If that is in fact correct I will revert :)